guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1280 @@
+
+#### Role#tags
+
+Tags for roles belonging to bots, integrations, or premium subscribers.
+
+### RoleManager
+
+#### RoleManager#botRoleFor
+
+`guild.roles.botRoleFor(user)`
+
+Gets the managed role a bot created when joining the guild, if any.
+
+#### RoleManager#edit
+
+`guild.roles.edit('12345678987654321', options)`
+
+Equivalent to `role.edit(options)`.
+
+#### RoleManager#premiumSubscriberRole
+
+`guild.roles.premiumSubscriberRole`
+
+Gets the premium subscriber (booster) role for the Guild, if any.
+
+### SelectMenuInteraction
+
+Provides gateway support for a `MessageComponentInteraction` coming from a select menu component.
+
+### StageChannel
+
+Provides API support for Stage Channels.
+
+### StageInstance
+
+Provides API support for Stage Instances. Stage instances contain information about live stages.
+
+### StageInstanceManager
+
+Provides API support for the bot to create, edit and delete live Stage Instances, and stores a cache of Stage Instances.
+
+### Sticker
+
+Provides API support for Discord Stickers.
+
+### StickerPack
+
+Provides API support for Discord Sticker packs.
+
+### TextChannel
+
+#### TextChannel#awaitMessageComponent
+
+A shortcut method to create a promisified `InteractionCollector` which resolves to a single `MessageComponentInteraction`.
+
+#### TextChannel#createMessageComponentCollector
+
+A shortcut method to create an `InteractionCollector` for components on a specific channel.
+
+#### TextChannel#setType
+
+`channel.setType('news')`
+
+Allows conversion between `TextChannel` and `NewsChannel`.
+
+#### TextChannel#threads
+
+Provides access to the `ThreadManager` for this channel.
+
+### ThreadChannel
+
+Provides API support for Thread Channels.
+
+### ThreadChannelManager
+
+Provides API support for the bot to create, edit and delete threads, and stores a cache of `ThreadChannels`.
+
+### ThreadMember
+
+Represent a member of a thread, and their thread-specific metadata.
+
+### ThreadMemberManager
+
+Provides API support for the bot to add and remove members from threads, and stores a cache of `ThreadMembers`.
+
+### Webhook
+
+#### Webhook#deleteMessage
+
+Webhooks can now delete messages that were sent by the Webhook.
+
+#### Webhook#editMessage
+
+Webhooks can now edit messages that were sent by the Webhook.
+
+#### Webhook#fetchMessage
+
+Webhooks can now fetch messages that were sent by the Webhook.
+
+#### Webhook#sourceChannel
+
+#### Webhook#sourceGuild
+
+Webhooks can now have a `sourceGuild` and `sourceChannel` if the message is being crossposted.
+
+### WelcomeChannel
+
+Represents the channels that can be seen in a Guild's `WelcomeScreen`.
+
+### WelcomeScreen
+
+Provides API support for a Guild's Welcome Screen.
+
+### Widget
+
+Represents a Guild's Widget.
+
+### WidgetMember
+
+Partial information about a guild's members stored in a Widget.
+
+### Util
+
+#### Formatters
+
+A number of new formatter functions are provided in the Util class, to easily handle adding markdown to strings.
```

Comment:
Something like
```suggestion
A number of new formatter functions have been moved from the Util class to the Formatters class, to easily handle adding markdown to strings.
```

Reply:
These never existed in Util - hence why I called them "new".